src/Tools/dotnet-monitor/Exceptions/ExceptionsOperation.cs

```diff
@@ -261,5 +271,26 @@ private static async Task WriteTextExceptionFormat(TextWriter writer, IException
                 await writer.WriteAsync(instance.Message);
             }
         }
+
+        public static string AssembleMethodName(string methodName, IList<string> parameterTypes)
```

Comment:
Inline this with the caller and write directly to the TextWriter; do not use StringBuilder

Reply:
Now it should actually be gone 😃